Screens/Settings.tsx

```diff
@@ -0,0 +1,537 @@
+        { value: 6, label: "Saturday", checked: false },
+    ],
+    deposit: "0.00",
+    regularHoursStart: "",
+    regularHoursEnd: "",
+    clientEmailNotifications: false,
+    clientSMSNotifications: false,
+};
+
+export default function Settings() {
+    const [sectionInfoVisible, setSectionInfoVisible] = useState({
+        businessName: false,
+        workingDays: false,
+        deposit: false,
+        regularHours: false,
+        clientNotifications: false,
+    });
+
+    const [noNotificationsChecked, setNoNotificationsChecked] = useState(true);
+
+    //TODO: update this to use the form values from db when those are availble rather than being hard coded
+
+    return (
+        <Formik
+            initialValues={settingsFormValues}
+            validationSchema={SettingsSchema}
+            onSubmit={(values) => console.log(values)}
+        >
+            {({
+                handleChange,
+                handleBlur,
+                handleSubmit,
+                setFieldValue,
+                values,
+                errors,
+                touched,
+            }) => (
+                <ScrollView style={styles.container}>
+                    <StatusBar />
+
+                    {/* ------------------------------Business Name----------------------------- */}
+
+                    <View style={styles.headerContainer}>
+                        <Text style={styles.header}>Business Name</Text>
+                        <TouchableOpacity
+                            onPress={() =>
+                                setSectionInfoVisible((pVal) => {
+                                    return {
+                                        ...pVal,
+                                        businessName: !pVal.businessName,
+                                    };
+                                })
+                            }
+                            hitSlop={checkboxHitSlop}
+                        >
+                            <Ionicons
+                                name="information-circle-sharp"
+                                style={styles.infoIcon}
+                            />
+                        </TouchableOpacity>
+                    </View>
+                    <Text
+                        style={[
+                            sectionInfoVisible.businessName
+                                ? styles.sectionInfoOpen
+                                : styles.sectionInfo,
+                        ]}
+                    >
+                        The name of your business as it will appear to your
+                        clients.
+                    </Text>
+
+                    <TextInput
+                        onChangeText={handleChange("businessName")}
+                        autoCapitalize="words"
+                        value={values.businessName}
+                        style={styles.input}
+                    ></TextInput>
+
+                    {errors.businessName && (
```

Comment:
You are using the `Errors` component in Login so you should probably use that here as well.